afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+ensom	-2
+lengsel	1
+vevstol	-1
+ruvet	-1
+truende	-2
+vevstoler	-1
+løs	-3
+taper	-3
+å miste	-3
+tap	-3
+tapt	-3
+elendig	-3
+elskelig	3
+kjærlig vennlighet	3
+lavest	-1
+lojal	3
+lojalitet	3
+flaks	3
+lukrative	3
+latterlig	-3
+galning	-3
+galninger	-3
+ligge på lur	-1
+lurer	-1
+luksus	2
+makaber	-2
+oppdiktet	-1
+storslått	3
+mishandling	-2
+påbudt, bindende	-1
+manipulert	-1
+manipulere	-1
+manipulasjon	-1
+uaktsomt drap	-3
+vidunder	3
+mesterverk	4
+saken	1
+saker	1
+moden	2
+gir mening	2
+meningsløs	-2
+medalje	3
+middelmådighet	-3
+melankoli	-2
+minneverdig	1
+trussel	-2
+truet	-2
+god	3
+fortryllende	3
+rot	-2
+rote til	-2
+metodisk	2
+tankeløs	-2
+mirakel	4
+munterhet	3
+dårlig oppførsel	-2
+ugagn	-1
+feilklassifiserer	-2
+stort	-3
+elendighet	-2
+feiltenning	-2
+feilinformert	-2
+mistolket	-2
+villede	-3
+forlegge	-2
+feilplassert	-2
+forlegger	-2
+feilprising	-3
+feilrapportering	-2
+feilrepresentere	-2
+feilaktig fremstilling	-2
+fremstillinger	-2
+misrepresented	-2
+uriktig	-2
+uriktig fremstilling	-2
+gå glipp av	-2
+savnet	-2
+mista	-2
+misforstå	-2
+misforståelse	-2
+misforstår	-2
+misforstått	-2
+misbruk	-2
+håne	-2
+gjøre narr av	-2
+modern	2
+modernisert	2
+moderniserer	2
+modernisering	2
+monopolisere	-2
+monopolisert	-2
+monopoliser	-2
+humørsyk	-1
+sturer	-1
+jævel	-5
+motivere	1
+motivert	2
+motiverende	2
+motivasjon	1
+sørge over	-2
+sørget	-2
+sørgmodig	-2
+gjørmete	-2
+mord	-2
+morder	-2
+morderisk	-3
+skummel	-2
+myte	-1
+naiv	-2
+narsissisme	-2
+ekkel	-3
+naturlig	1
+trengende	-2
+negativ	-2
+negativitet	-2
+forsømmelse	-2
+forsømt	-2
+neglisjerer	-2
+nerver	-1
+nervøs	-2
+nervøst	-2
+nei	-1
+ikke moro	-3
+edel	2
+edleste	2
+støyende	-1
+ikke-godkjent	-2
+nybegynner	-2
+ikke bra	-2
+jobber ikke	-3
+bemerkelsesverdig	2
+merkbar	2
+notorisk	-2
+roman	2
+nummen	-1
+pleie	2
+nøtter	-3
+utslette	-2
+utslettet	-2
+ytterst ubehagelig	-3
+obskøn	-2
+obskønitet	-2
+besatt	2
+foreldet	-2
+hindringer	-2
+sta	-2
+hinder	-2
+hindrer	-1
+merkelig	-2
+lovbrudd	-2
+fornærme	-2
+lovbryteren	-2
+uakseptable	-2
+støtende	-2
+offensivt	-2
+oks	2
+illevarslende	3
+en gang i livet	3
+muligheter	2
+mulighet	2
+undertrykt	-2
+undertrykkelse	-2
+undertrykkelser	-2
+undertrykkende	-2
+optimisme	2
+optimistisk	2
+utstøtt	-2
+au	-2
+utbrudd	-2
+ramaskrik	-2
+utmanøvrert	-2
+mindretall	-2
+opprør	-3
+oppsøk	2
+fremragende	5
+overbelastning	-1
+oversett	-1
+overbeskyttende	-2
+erobret	-2
+reagere	-2
+overreagerte	-2
+reagerer	-2
+overreaksjon	-2
+sell	-2
+sells	-2
+forglemmelse	-1
+forenkling	-2
+unyansert	-2
+overforenkle	-2
+solgt	-2
+overdriv	-2
+overvekt	-1
+overspent	-3
+smertelig	-2
+smertefull	-2
+panikk	-2
+fikk panikk	-3
+paradis	3
+paradoks	-1
+benådet	2
+tilgivelse	2
+benådninger	2
+lidenskap	1
+lidenskapelig	2
+passiv	-1
+passivt	-1
+patetisk	-2
+betale	-1
+fred	2
+fredelig	2
+straffe	-2
+straffet	-2
+straffer	-2
+penalising	-2
+straff	-2
+tankefull	-1
+perfekt	3
+perfeksjonert	2
+fullkommenhet	3
+perfeksjonerer	2
+peril	-2
```

Comment:
Is this a Norwegian word?

Reply:
The list was translated with Google Translate, and the list I submitted is compiled from 4 lists:
* words translated to norwegian
* english words that could not be translated
* new words I manually added
* a "stoplist" of words to be removed from the final list

I have left the untranslatable english words in the list as I thought it couldn't harm and sometimes english is mised in with norwegian anyway. 

I can resubmit the list without the english words if you prefer.